src/autoplex/data/phonons/flows.py

```diff
@@ -649,6 +649,27 @@ def make_from_ml_model(
         elif ml_model == "J-ACE":
             raise UserWarning("No atomate2 ACE.jl PhononMaker implemented.")
 
+        elif ml_model == "NEP":
+            if calculator_kwargs is None:
+                calculator_kwargs = {"model_filename": str(potential_file)}
+
+            ml_prep = ml_phonon_maker_preparation(
+                bulk_relax_maker=ForceFieldRelaxMaker(
+                    relax_cell=True,
+                    relax_kwargs={"interval": 500},
+                    force_field_name="NEP",
+                ),
+                phonon_displacement_maker=ForceFieldStaticMaker(
+                    name="nep phonon static",
+                    force_field_name="NEP",
+                ),
+                static_energy_maker=ForceFieldStaticMaker(
+                    force_field_name="NEP",
+                ),
```

Comment:
Once https://github.com/autoatml/autoplex/pull/333 is merged, this has to be set to None
```suggestion
                static_energy_maker=None,
```

Reply:
Hi @QuantumChemist , maybe better to do it in your PR itself. I think this PR is now ready to be merged at the point. The other PR may take longer I think ?

Reply:
of course, will adjust it in the other PR then 😃